app/models/mixins/purging_mixin.rb

```diff
@@ -21,6 +21,20 @@
 #     the ids of the records that have just been deleted, and should purge any
 #     other records associated with those ids.
 #   ? can this be auto defined looking at relations and destroy / delete
+#
+# For purge_by_remaining, the mixee must provide the following methods:
+#
+#   @return [String] Sql with the resource that is referenced by this table.
+#   def purge_remaining_foreign_key
+#     return "foreign_id"
```

Comment:
This doesn't match how you have to actually write it in code as you've done elsewhere in this PR.